open-craft/opencraft
# Increase servername prefix length to 30 chars

Some instances with very similar names are generating the same servernames (Ironwood and Juniper periodic instances). This aims to be a fix for those. Keeping a reasonable limit to prevent excessively large hostnames, but a 10 character increase should do fine.

**JIRA tickets**: [SE-3484](https://tasks.opencraft.com/browse/SE-3484)

**Testing instructions**:

1. Make sure the unit tests pass (in the CI, or via `make test`)
2. Verify that stage works as expected. [This](https://stage.manage.opencraft.com/instance/5290/edx-appserver/2614/) is a sample instance generated using this branch.

**Reviewers**
